src/proto/grpc/status/README

```diff
@@ -0,0 +1,2 @@
+The status.proto file is copied from
+https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto.
```

Comment:
Sounds like a good idea! I will try to add a sanity check in another PR. I sent an email to the owner about the stability of the file and have yet received a reply. But it seems to me the file is pretty stable (no API change ever) at the moment.